app/assets/javascripts/models/beta/story.js

```diff
@@ -74,11 +74,20 @@ export const update = (story, projectId, options) => {
   const newStory = changeCase.snakeKeys(serialize(story), { recursive: true, arrayRecursive: true });
 
   return httpService
-    .put(`/projects/${projectId}/stories/${story.id}`, {story: newStory})
+    .put(`/projects/${projectId}/stories/${story.id}`, { story: newStory })
     .then(({ data }) => changeCase.camelKeys(data, { recursive: true, arrayRecursive: true }))
     .then(({ story }) => deserialize(story, options));
 };
 
+export const post = (story, projectId) => {
+  const newStory = changeCase.snakeKeys(serialize(story), { recursive: true, arrayRecursive: true });
+
+  return httpService
+    .post(`/projects/${projectId}/stories`, { story: newStory })
+    .then(({ data }) => changeCase.camelKeys(data, { recursive: true, arrayRecursive: true }))
```

Comment:
We should extract this logic and reuse. Also, camelization of the keys should be a responsibility of the `deserialize` method, what do you think? We should have only `then(({ data }) => deserialize(data.story))` here.